pkg/dom0-ztools/rootfs/etc/containerd/config.toml

```diff
@@ -1,33 +1,23 @@
-version = 2
-state = "/run/containerd"
+version = 3
 root = "/persist/containerd-system-root"
 disabled_plugins = [
     "io.containerd.snapshotter.v1.btrfs",
     "io.containerd.snapshotter.v1.aufs",
     "io.containerd.internal.v1.opt",
     "io.containerd.internal.v1.tracing",
-    "io.containerd.tracing.processor.v1.otlp"
+    "io.containerd.tracing.processor.v1.otlp",
+    "io.containerd.runtime.v1.linux",
```

Comment:
Is this sensitive to the extra `,`? I know json is, go is not, and everything else is confusing.

Reply:
it seems it is working. I generated configs using containerd itself so this is added by containerd